modules/gdscript/gdscript.cpp

```diff
@@ -892,13 +891,19 @@ bool GDScript::_get(const StringName &p_name, Variant &r_ret) const {
 				return true;
 			}
 		}
-
 		{
 			HashMap<StringName, MemberInfo>::ConstIterator E = top->static_variables_indices.find(p_name);
 			if (E) {
 				if (E->value.getter) {
 					Callable::CallError ce;
-					r_ret = const_cast<GDScript *>(this)->callp(E->value.getter, nullptr, 0, ce);
+					const GDScript::MemberInfo *member = &E->value;
```

Comment:
Below you use the name `static_member` for static variables. I suggest to do the same here, for consistency.